game.py

```diff
@@ -98,27 +127,28 @@ def declare_winner(self):
 
     def play_game(self):
         self.initialize_deck()
-        number_of_players = self.get_number_of_players()
-        self.initialize_players_data(number_of_players)
+        self.get_number_of_players()
+        self.initialize_players_data()
         self.put_initial_card_on_table()
-        player_turn = 0
-        print(f"\n *** Game started *** \n")
+        print(f"\n *** Game has started *** \n")
         self.print_game_status()
+        if self.card_on_table.is_stop_card:
```

Comment:
is there a way you can incorporate this into the game while loop?
i assume the behavior of a stop card is the same regardless of whether the player is the initial one or not?